subprojects/parseq/src/main/java/com/linkedin/parseq/Task.java

```diff
@@ -1403,7 +1402,7 @@
    * @return a new task that will submit the callable to given executor and complete
    * with result returned by that callable
    */
-  public static <T> Task<T> blocking(final String name, final Callable<? extends T> callable, final Executor executor) {
+  public static <T> Task<T> runInExecutor(final String name, final Callable<? extends T> callable, final Executor executor) {
```

Comment:
this is backward incompatible change. why do we need to do this? can we add additional API and deprecate this or just update the java doc to make it less confusing.

Reply:
The function with same name still exists and was annotated as deprecated. 

Isn't that what you meant?

Reply:
oops didn't see below. diff showed replacing name. please ignore.